debezium-core/src/main/java/io/debezium/connector/common/CdcSourceTaskContext.java

```diff
@@ -42,7 +43,8 @@ public CdcSourceTaskContext(CommonConnectorConfig config,
                                 Map<String, String> customMetricTags,
                                 Supplier<Collection<? extends DataCollectionId>> collectionsSupplier) {
         this.connectorType = config.getContextName();
-        this.connectorName = config.getLogicalName();
+        this.connectorLogicalName = config.getLogicalName();
```

Comment:
This is now quite confusing. Connector name reflects the connector type itself and connector logical name reflects - well the topic prefix aka logical name of a connector instance. I believe we need to differentiate between those. Maybe use something like `getConnectorInstanceLogicalName()`?

Reply:
and well there is a long list o places where `connectorName` is used...